pkgs/development/libraries/wlroots/0.15.nix

```diff
@@ -1,7 +1,28 @@
-{ lib, stdenv, fetchFromGitLab, meson_0_60, ninja, pkg-config, wayland-scanner
-, libGL, wayland, wayland-protocols, libinput, libxkbcommon, pixman
-, xcbutilwm, libX11, libcap, xcbutilimage, xcbutilerrors, mesa
-, libpng, ffmpeg, xcbutilrenderutil, seatd, vulkan-loader, glslang
+{ lib
+, stdenv
+, fetchFromGitLab
+, ffmpeg
+, glslang
+, libGL
+, libX11
+, libcap
+, libinput
+, libpng
+, libxkbcommon
+, mesa
+, meson
+, ninja
+, pixman
+, pkg-config
+, seatd
+, vulkan-loader
+, wayland
+, wayland-protocols
+, wayland-scanner
+, xcbutilerrors
+, xcbutilimage
+, xcbutilrenderutil
+, xcbutilwm
```

Comment:
Not a blocker but I hate those refactorings in unrelated commits (it's a bad commit practice). We should either reformat all of Nixpkgs at once or packages individually. A commit like https://github.com/NixOS/nixpkgs/pull/153082/commits/3d74e5725990dee4b934a8a1be8f8aa615c5da9e ("wlroots 0.15: change meson attribute - Meson 0.60 is the default now") shouldn't contain such unrelated changes.

Reply:
Understandable.

I prefer to do such code refactorings among other things because they did not change the semantics of the code.